.github/workflows/release.yml

```diff
@@ -771,6 +796,8 @@ jobs:
     - name: Build (without installer)
       if: needs.source.outputs.is_tag != 'true'
       shell: bash
+      env:
+        WIN_CODESIGN_CN: ${{ secrets.WIN_CERTIFICATE_COMMON_NAME }}
```

Comment:
`CN`? Why not just `COMMON_NAME`?

Reply:
Mainly because it's referred to as "CN=OpenTTD Distribution Ltd" in the certificate itself, it's the usual abbreviation.

Reply:
I mean that the left side uses CN and the right side COMMON_NAME :)